### PR TITLE
Go getting started page

### PR DESCRIPTION
**Pull request by efirs**

### CODE REVIEW

File: docs/quickstart/with-go.md [hunk 1]
```diff
@@ -1,4 +1,293 @@
...
-possible using the Go.
+:::tip
+⏱ This guide should require less than 7 minutes
+
+:::
+
+This guide shows how to integrate Tigris with the backend of microservice architecture
+application, using simplified eCommerce service implementation.
+
+The guide consists of the following parts:
```
Comment (ovaistariq) [inline docs/quickstart/with-go.md:11]: We don't need these lines as navigation based on headings are automatically generated

File: docs/quickstart/with-go.md [hunk 2]
```diff
@@ -1,4 +1,293 @@
...
+
+Once this command has completed, Tigris server will be available on port
+`8081`.
+
+## Setting up and starting the service
+
+### Clone the starter application code repository
+
+```shell
+git clone https://github.com/tigrisdata/tigris-starter-go.git
```
Comment (ovaistariq) [inline docs/quickstart/with-go.md:66]: Can we have the user download a tarball instead? That way they can use the code in their own git repo

File: docs/quickstart/with-go.md [hunk 3]
```diff
@@ -1,4 +1,293 @@
...
+```shell
+curl -X POST localhost:8080/products/create -H 'Content-Type: application/json' \
+	 -d '{"id":1,"Name":"Avocado","Price":10,"Quantity":5}'
+curl -X POST localhost:8080/products/create -H 'Content-Type: application/json' \
+	 -d '{"id":2,"Name":"Gold","Price":3000,"Quantity":1}'
+```
+
+### Place some orders
+
+#### Low balance
```
Comment (ovaistariq) [inline docs/quickstart/with-go.md:104]: This is really good and would be better suited in a guide. We will add a separate section for guides and can add this detailed version as a guide. 

For the QuickStart let's keep it short and sweet.

File: docs/quickstart/with-go.md [hunk 4]
```diff
@@ -1,4 +1,293 @@
...
+```
+
+### Compile and start the application
+```shell
+go build .
+./tigris-starter-go
+```
+
+## Testing the service
+
```
Comment (ovaistariq) [inline docs/quickstart/with-go.md:77]: Could we also have a swagger url similar to what we have for Java QuickStart that lists all the endpoints and allows the user to test the service via the UI?

File: docs/quickstart/with-go.md (hunk 1 above)
Comment (efirs) [reply docs/quickstart/with-go.md:11]: Yeah, I thought it was only in dropdown, but if I zoom out it becomes visible on the right. removing.

File: docs/quickstart/with-go.md (hunk 4 above)
Comment (efirs) [reply docs/quickstart/with-go.md:77]: I'll check if there is an easy way, but I didn't find anything at first glance.

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**